src/pages/[platform]/build-a-backend/functions/examples/event-source/index.mdx

```diff
@@ -0,0 +1,202 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Lambda event source',
```

Comment:
can we change this title to be a bit more descriptive? maybe just "Event handlers"?